pysat/instruments/madrigal_pandas.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-.
+"""Supports generalized access to Madrigal Data.
+
+To use this routine, you need to know both the Madrigal Instrument code
+as well as the data tag numbers that Madrigal uses to uniquely identify
+data sets. Using these codes, the madrigal_methods.py routines will
+be used to support downloading and loading of data.
+
+
+Downloads data from the JRO Madrigal Database.
```

Comment:
Do we want to default to the JRO database?

Reply:
It doesn't actually default to JRO Madrigal, as far as I can tell. I updated the comment.